GoogleDataTransport/GDTCORLibrary/GDTCOREventMetricsCounter.m

```diff
@@ -40,8 +42,7 @@ + (instancetype)counterWithEvents:(NSArray<GDTCOREvent *> *)events
   NSMutableDictionary<NSString *, GDTCORDroppedEventCounter *> *eventCounterByMappingID =
       [NSMutableDictionary dictionary];
 
-  for (GDTCOREvent *event in events) {
-    // TODO(ncooke3): Should I use an autorelease pool?
```

Comment:
I did some profiling in a unit test where I had 1000 events. Wrapping the loop contents in an `@autoreleasepool` actually increased the memory footprint slightly. Removing this TODO as it doesn't seem like a worthy optimization.